source/kubernetes/minio-kubernetes-overview.rst

```diff
@@ -1,27 +1,725 @@
+
+- ``storageClassName`` *must* be empty.
+
+- The ``path`` directory *must* be empty.
+
+Issue the ``kubectl get PV`` command to validate the created PVs:
+
+.. code-block:: shell
+   :class: copyable
+
+   kubectl get PV
+
+3) Create a Namespace for the MinIO Tenant
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the ``kubectl create namespace`` command to create a namespace for
+the MinIO Tenant:
+
+.. code-block:: shell
+   :class: copyable
+
+   kubectl create namespace minio-tenant-1
+
+4) Create the MinIO Tenant
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the :mc-cmd:`kubectl minio tenant create` command to create the MinIO
+Tenant.
+
+The following example creates a 4-node MinIO deployment with a
+total capacity of 16Ti across 16 drives.
+
+.. code-block:: shell
+   :class: copyable
+
+   kubectl minio tenant create    \
+     --name      minio-tenant-1   \
+     --servers   4                \
+     --volumes   16               \
+     --capacity  16Ti             \
+     --namespace minio-tenant-1
+
+The following table explains each argument specified to the command:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+   :width: 100%
+
+   * - Argument
+     - Description
+
+   * - :mc-cmd-option:`~kubectl minio tenant create name`
+     - The name of the MinIO Tenant which the command creates.
+
+   * - :mc-cmd-option:`~kubectl minio tenant create servers`
+     - The number of :mc:`minio` servers to deploy across the Kubernetes 
+       cluster.
+
+   * - :mc-cmd-option:`~kubectl minio tenant create volumes`
+     - The number of volumes in the cluster. :mc:`kubectl minio` determines the
+       number of volumes per server by dividing ``volumes`` by ``servers``.
+
+   * - :mc-cmd-option:`~kubectl minio tenant create capacity`
+     - The total capacity of the cluster. :mc:`kubectl minio` determines the 
+       capacity of each volume by dividing ``capacity`` by ``volumes``.
+
+   * - :mc-cmd-option:`~kubectl minio tenant create namespace`
+     - The Kubernetes namespace in which to deploy the MinIO Tenant.
+
+If :mc-cmd:`kubectl minio tenant create` succeeds in creating the MinIO Tenant,
+the command outputs connection information to the terminal. The output includes
+the access key and secret key required for connecting to the MinIO server
+*and* the MinIO Console.
+
+.. code-block:: shell
+   :emphasize-lines: 1-3, 6-8
+   
+   Tenant
+   Access Key: 999466bb-8bd6-4d73-8115-61df1b0311f4
+   Secret Key: f8e5ecc3-7657-493b-b967-aaf350daeec9
+   Version: minio/minio:RELEASE.2020-09-26T03-44-56Z
+   ClusterIP Service: minio-tenant-1-internal-service
+
+   MinIO Console
+   Access Key: e9ae0f3f-18e5-44c6-a2aa-dc2e95497734
+   Secret Key: 498ae13a-2f70-4adf-a38e-730d24327426
+   Version: minio/console:v0.3.14
+   ClusterIP Service: minio-tenant-1-console
+
+Kubernetes users with permission to perform ``kubectl get secrets <secret>`` can
+view the secret contents and extract the access and secret key. The keys are
+Base64 encoded.
```

Comment:
If we're talking about the internal secret that is created, we should tell the name of the secret as well.

Reply:
Good point - I think we prefix the namespace + tenant name or something . Will check.

Reply:
Added a sample of what the output looks like.